scripts/run_eval_vllm_gaudi.sh

```diff
@@ -0,0 +1,9 @@
+export host_ip=$(hostname -I | awk '{print $1}')
+export LLM_ENDPOINT_PORT=8010
+export DATA_PATH="~/.cache/huggingface"
+export LLM_ENDPOINT="http://${host_ip}:${LLM_ENDPOINT_PORT}/v1"
+export HF_HOME=$DATA_PATH
+
+for task in "recall" "rag"; do
+    python eval.py --config configs/${task}_vllm.yaml --endpoint_url $LLM_ENDPOINT --overwrite --batch_mode multi_thread --no_cuda
```

Comment:
here you are using `batch_mode = multi_thread`, but in the `eval.py`, you only added `args.batch_mode == "batch_api"` on line 80. If you want to enable batching for the Intel models, you would also need to change the `generate_batch` function under `TgiVllmModel` so that it always uses threading instead of the current batch file default.